docs/technical-documentation/flysystem.md

```diff
@@ -14,13 +14,13 @@ The Drupal Flysystem module uses [flysystem stream wrappers](https://github.com/
 
 ### The Plugin and Adapter
 
-Islandora 8 implements a [Flysystem adapter](https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/src/Flysystem/Adapter/FedoraAdapter.php) and a [Drupal Flysystem plugin](https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/src/Flysystem/Fedora.php). The Flysystem adapter acts as an intermediary between the Flysystem filesystem API and Fedora, translating requests and responses between them. The adapter interacts with Fedora using an instance of the [Chullo Fedora API](/Islandora-CLAW/chullo/blob/master/src/IFedoraApi.php) it receives from the Drupal Flysystem plugin. The Drupal Flysystem plugin's main responsibility is to instantiate the Chullo Fedora API object with the proper authentication and pass it to the Flysystem adapter. To authenticate with Fedora the plugin adds a [handler](http://docs.guzzlephp.org/en/stable/handlers-and-middleware.html) to the Chullo's [Guzzle](http://docs.guzzlephp.org) client which adds a [JWT](https://jwt.io/) authentication header to each request. <!-- After https://github.com/Islandora-CLAW/islandora/pull/119 is merged:  To authenticate with Fedora the plugin adds a handler to the Chullo's Guzzle client with a copy of the site's JWT authentication object so that each Fedora request can generate its own Bearer token. -->
+Islandora 8 implements a [Flysystem adapter](https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/src/Flysystem/Adapter/FedoraAdapter.php) and a [Drupal Flysystem plugin](https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/src/Flysystem/Fedora.php). The Flysystem adapter acts as an intermediary between the Flysystem filesystem API and Fedora, translating requests and responses between them. The adapter interacts with Fedora using an instance of the [Chullo Fedora API](/Islandora-CLAW/chullo/blob/main/src/IFedoraApi.php) it receives from the Drupal Flysystem plugin. The Drupal Flysystem plugin's main responsibility is to instantiate the Chullo Fedora API object with the proper authentication and pass it to the Flysystem adapter. To authenticate with Fedora the plugin adds a [handler](http://docs.guzzlephp.org/en/stable/handlers-and-middleware.html) to the Chullo's [Guzzle](http://docs.guzzlephp.org) client which adds a [JWT](https://jwt.io/) authentication header to each request. <!-- After https://github.com/Islandora-CLAW/islandora/pull/119 is merged:  To authenticate with Fedora the plugin adds a handler to the Chullo's Guzzle client with a copy of the site's JWT authentication object so that each Fedora request can generate its own Bearer token. -->
```

Comment:
This looks to have been there before but the URL for the `Chullo Fedora API` is missing `https:/` and so the link is broken.

Also the commented out text at the bottom states
> After https://github.com/Islandora-CLAW/islandora/pull/119 is merged:

It is merged, so this can (I assume) by revealed.

Also there were some CLAWs in here.
 ```suggestion
Islandora 8 implements a [Flysystem adapter](https://github.com/Islandora/islandora/blob/8.x-1.x/src/Flysystem/Adapter/FedoraAdapter.php) and a [Drupal Flysystem plugin](https://github.com/Islandora/islandora/blob/8.x-1.x/src/Flysystem/Fedora.php). The Flysystem adapter acts as an intermediary between the Flysystem filesystem API and Fedora, translating requests and responses between them. The adapter interacts with Fedora using an instance of the [Chullo Fedora API](https://github.com/Islandora/chullo/blob/main/src/IFedoraApi.php) it receives from the Drupal Flysystem plugin. The Drupal Flysystem plugin's main responsibility is to instantiate the Chullo Fedora API object with the proper authentication and pass it to the Flysystem adapter. To authenticate with Fedora the plugin adds a [handler](http://docs.guzzlephp.org/en/stable/handlers-and-middleware.html) to the Chullo's [Guzzle](http://docs.guzzlephp.org) client which adds a [JWT](https://jwt.io/) authentication header to each request. To authenticate with Fedora the plugin adds a handler to the Chullo's Guzzle client with a copy of the site's JWT authentication object so that each Fedora request can generate its own Bearer token.
```

Reply:
requiring the full de-clawing of the documentation right now would also require re-writing and also updating a large number of formerly-useful pages... that's an ongoing process, can we do the master-main rename first?

Reply:
I have no problem not doing a full de-clawing, I just noted the changes on this line as the PR already touched it.

Reply:
phew. ty. Yeah there are a number of pages in here that need love. I'm currently looking at "hacking on islandora".